modules/core/src/main/java/org/apache/ignite/mxbean/IgniteMXBean.java

```diff
@@ -380,23 +376,30 @@
      */
     @MXBeanDescription("Pings node with given node ID to see whether it is alive. " +
         "Returns whether or not node is alive.")
-    @MXBeanParametersNames(
-        "nodeId"
-    )
-    @MXBeanParametersDescriptions(
-        "String presentation of node ID. See java.util.UUID class for details."
-    )
+    @MXBeanParametersNames("nodeId")
+    @MXBeanParametersDescriptions("String presentation of node ID. See java.util.UUID class for details.")
     public boolean pingNode(String nodeId);
 
+    /** Activates cluster. */
+    @MXBeanDescription("Execute activation process.")
+    @MXBeanParametersNames("active")
+    public void activate();
+
+    /** Deactivates cluster. */
+    @MXBeanDescription("Execute deactivation process.")
+    @MXBeanParametersNames("force")
```

Comment:
We should add MXBeanParametersDescriptions

Reply:
Fixed

Reply:
@nizhikov , please review IgniteMXBean. We have to put flag 'force' somewhere for the deactivation. Current choice is

`IgniteMXBean#clusterState(String state) -> clusterState(String state, boolean force)`

Usually one has to support backward compatibility of an Interface. But this is an JMX. I don't think it is unsafe to add single param. Otherwise we need to implement additional method like

` clusterState(String state, boolean force)
or
deactivate(boolean force)`

whereas the mxbean already has several methods for the cluster state:
`
void active(boolean active)
boolean active()
void clusterState(String state)
String clusterState()
`

Don't think one more would be handy;

Also, should we declare 
`IgniteMXBean#active(boolean active)`
deprecated too.

Reply:
> But this is an JMX. I don't think it is unsafe to add single param. 

Why do you think so?

Reply:
> > But this is an JMX. I don't think it is unsafe to add single param.
> 
> Why do you think so?

An automated, external system like Zabbix of something must use exposed bean to be afraid of it. Manual method invocation in the console is quite obvious. Just requires to say extra ‘yes’ or ‘no’ for the new argument. Direct bean usage within the code isn't reasonable except the tests.

Reply:
JMX bean is a public API, so we should preserve backward compatibility.

Reply:
> JMX bean is a public API, so we should preserve backward compatibility.

Ok.

And

>> Also, should we declare
>> `IgniteMXBean#active(boolean active)`
>> deprecated too?

Reply:
I think it should be discussed on the dev-list